src/unity/schemas/Organization.yml

```diff
@@ -1,23 +1,37 @@
 type: object
 properties:
-  idpeId:
-    type: string
-    description: idpe ID of the org
   id:
-    type: number
-    description: quartz ID of the org
+    type: string
+    description: ID of the org
   name:
     type: string
     description: name of the org
-  region:
+  description:
+    type: string
+    description: the description of the organization
+  creationDate:
+    type: string
+    format: date-time
+    description: the date the organization was created
+  regionCode:
+    type: string
+    description: the code associated with the region
```

Comment:
```suggestion
    description: the canonical code for the region where the organization is hosted
```

Reply:
Updated in https://github.com/influxdata/openapi/pull/304/commits/0346df21a2f03e7163b1469b1c957660c3903cfc